### PR TITLE
Metrics: Remove iperf2 tests

### PR DESCRIPTION
**Pull request by GabyCT**
Remove iperf2 metrics as they are already implemented
using iperf3 (see #781)

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: tests/metrics/network/network-metrics.sh [hunk 1]
```diff
@@ -19,8 +19,7 @@
 #  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #
 # Description:
-#  Test inter (docker<->docker) network bandwidths and jitters
-# using iperf2
+#  Test inter (docker<->docker) bidirectional network bandwidth using iperf2
```
Comment (jcvenegas) [inline tests/metrics/network/network-metrics.sh]: is iperf2 ok  in the description ?

**Comment by jcvenegas** [comment]
just one question but
lgtm

File: tests/metrics/network/network-metrics.sh (hunk 1 above)
Comment (GabyCT) [inline tests/metrics/network/network-metrics.sh]: @jcvenegas yes it is ok, we are removing the rests of the test keeping only the bidirectional one :)

**Comment by chavafg** [comment]
qa-passed

**Pull request merged**